be/src/olap/tablet_manager.cpp

```diff
@@ -301,6 +301,11 @@ OLAPStatus TabletManager::create_tablet(const TCreateTabletReq& request,
         LOG(WARNING) << "fail to create tablet. res=" << res;
     }
 
+    // call calculate_cumulative_point() manually.
+    // If not, cumulative_point will set to minus one
+    // when init tablet during create tablet.
+    RETURN_NOT_OK(tablet->calculate_cumulative_point());
+
```

Comment:
If calculate method is called in two cases, it's better to call it during cumulative compaction.